Controllers/EmployeeController.cs

```diff
@@ -16,12 +16,105 @@ namespace TodoApi.Controllers
     public class EmployeeController : ControllerBase
     {
         private readonly ApiDbContext _context;
+        private readonly string connectionString = "Server=myserver;Database=mydb;User Id=admin;Password=password;";
 
 
         public EmployeeController(ApiDbContext context)
         {
             _context = context;
         }
+        
+       [HttpGet]
+        public async Task<IActionResult> GetEmployeeBlocking()
+        {
+            var employee = _context.Employees.FindAsync(1).Result; // Blocking async call
+            return Ok(employee);
```

Comment:
Ensure proper disposal of `ApiDbContext` instances to avoid potential memory leaks and connection issues.